appveyor.yml

```diff
@@ -23,5 +23,7 @@ install:
 
 test_script:
   - cd %APPVEYOR_BUILD_FOLDER%
+  - php tests\Fixtures\app\console cache:clear
```

Comment:
I wish I know the real reason... Both suites works without issue independently but when they are run without clearing the cache, in the second suite `NelmioApiDocBundle` was enabled for now real reason and trigger deprecation notices. (`Enabling the NelmioApiDocBundle integration has been deprecated in 2.2 and will be removed in 3.0. NelmioApiDocBundle 3 has native support for API Platform.`)
I've been fighting with this for a while on appveyor and finally looked at travis config
https://github.com/api-platform/core/blob/7dbd20775f02556bd09c8715ac1055e51dc38597/.travis.yml#L45-L53

It might be related to the way that I'm booting the kernel for the test...
😞